src/Fantomas/Format.fs

```diff
@@ -5,15 +5,16 @@ open System.IO
 open Fantomas.Core
 
 type FormatResult =
-    | Formatted of filename: string * formattedContent: string
-    | Unchanged of filename: string
+    | Formatted of filename: string * formattedContent: string * profileInfos: ProfileInfos option
```

Comment:
I don't think `ProfileInfos` would be plural here. `ProfileInfo` seems more correct.

Reply:
Done